pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -171,7 +168,7 @@ type ImageTag struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
 
-	TagName string `json:"tagName,omitempty"`
+	Image *Image `json:"image,omitempty"`
```

Comment:
```suggestion
	Tags []string `json:"tags,omitempty"`
```
The other fields of image, namely Digest don't seem to have any use, lets just have the one field `Tags`

Reply:
I changed to Image *Image with image/tags now having to call the create so now the requestImage can be created if it does not exist

```
func (t *TagStrategy) ImageTag(ctx context.Context, namespace, imageName string, requestImage *apiv1.Image) (*apiv1.Image, error) {
	image := &apiv1.Image{}
	imageList := &apiv1.ImageList{}
	err := t.client.Get(ctx, router.Key(namespace, imageName), image)
	if apierror.IsNotFound(err) {
		err = t.client.Create(ctx, requestImage)
		if err != nil {
			return image, err
		}
```

Reply:
This is a side effect of the fact i've rewritten things.  We don't want the user to be able to create new Image objects, but right now you have to create images from the client.  So I guess just leave it as is and I'll refactor it in my PR.

Reply:
okay 👍